src/bar.jl

```diff
@@ -48,15 +48,9 @@ grouped_xy(y::AbstractArray) = 1:size(y,1), y
     end
 
     # compute fillrange
-    y, fr = isstack ? groupedbar_fillrange(y) : (y, get(plotattributes, :fillrange, nothing))
```

Comment:
So how are the y positions and fillranges for stacked bars calculated now?

Reply:
It's right below:
```julia
fill_bottom = min(minimum(y) / 100, the_ylims[1])
```

Reply:
Happy to have more test cases though

Reply:
But they are different for `:stack`ed groupedbars, which is calculated in `groupedbar_fillrange`. These changes only cover the `:dodge` case.

Reply:
```julia
using StatsPlots

x = [
    1 2 2
    3 2 1
]

groupedbar(x, bar_position = :stack)
```
With this PR:
![stackedbar](https://user-images.githubusercontent.com/16589944/89773031-6c810100-db03-11ea-980d-a2a63222cf9c.png)

On master:
![stackedbar_master](https://user-images.githubusercontent.com/16589944/89773212-c97cb700-db03-11ea-8359-8dde0299db1b.png)

Reply:
should be fixed now